qt/python/mantidqtinterfaces/mantidqtinterfaces/Engineering/gui/engineering_diffraction/tabs/gsas2/call_G2sc.py

```diff
@@ -178,11 +178,10 @@ def export_lattice_parameters(temp_save_directory, name_of_project, project):
 d_spacing_min = inputs_dict["d_spacing_min"]
 number_of_regions = inputs_dict["number_of_regions"]
 
-
 """Call GSASIIscriptable"""
 import_path = None
 try:
-    import_path = os.path.join(path_to_gsas2, "GSASII")
+    import_path = os.path.join(path_to_gsas2, "GSAS-II", "GSASII")
```

Comment:
I have installed latest GSAS-II for windows from https://subversion.xray.aps.anl.gov/trac/pyGSAS/wiki/InstallWindows. I am getting below error when `Refine in GSAS` is clicked. 
Note, in my installation `gsas2full\GSAS-II\GSASII` path does not exist and it is `gsas2full\GSASII` directory to find `GSASIIscriptable.py`

```
GSAS-II call failed with error: Traceback (most recent call last):
  File "C:\Codes\mantid_for_Testing\qt\python\mantidqtinterfaces\mantidqtinterfaces\Engineering\gui\engineering_diffraction\tabs\gsas2\call_G2sc.py", line 186, in <module>
    import GSASIIscriptable as G2sc
ModuleNotFoundError: No module named 'GSASIIscriptable'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Codes\mantid_for_Testing\qt\python\mantidqtinterfaces\mantidqtinterfaces\Engineering\gui\engineering_diffraction\tabs\gsas2\call_G2sc.py", line 188, in <module>
    raise ImportError(f"GSAS-II was not found at {import_path}")
ImportError: GSAS-II was not found at C:/Users/kli94267/AppData/Local/gsas2full\GSAS-II\GSASII
```

Reply:
Yes sorry, I had to re-install GSAS and I think in later versions the path has changed - but I didn't mean to include this change in the PR as it is currently working on IDAaaS. I'll change it back, but for now if you could correct this path and continue with the testing that'd be great thanks!